ddc_client.py

```diff
@@ -59,9 +59,9 @@ def start(self,):
           # upgrade components if necessary
           need_restart = False
           for xml_upgrade in xml_response.findall("upgrades/upgrade"):
-            type = xml_upgrade.get("type")
+            ttype = xml_upgrade.get("type")
```

Comment:
@nilnilnil "type" is a built-in functions  https://docs.python.org/3/library/functions.html#type